cmake/std/PullRequestLinuxDriverTest.py

```diff
@@ -42,706 +40,221 @@
-                module('swap', unl_mod, load_mod)
-
-    if 'OMPI_CC' in l_environMap:
-        l_environMap['OMPI_CC'] = os.environ.get('CC', '')
-
-    if 'OMPI_FC' in l_environMap:
-        l_environMap['OMPI_FC'] = os.environ.get('FC', '')
-
-    for key, value in l_environMap.items():
-        if key in os.environ:
-            # we are assuming these are paths to be prepended
-            os.environ[key] = str(value) + os.pathsep + os.environ[key]
-        else:
-            os.environ[key] = str(value)
-
-    confirmGitVersion()
-
-    print ("Environment:\n")
-    print("  pwd = {cwd}".format(cwd=os.getcwd()))
-    print("")
-    for key in os.environ:
-        print(key + ' = ' + os.environ[key],
-              file=sys.stdout)
-
-    print("\n"+"="*90)
-    print(module('list'))
+    arguments = parser.parse_args()
 
-    return None
+    # Type conversions
+    arguments.max_cores_allowed    = int(arguments.max_cores_allowed)
+    arguments.num_concurrent_tests = int(arguments.num_concurrent_tests)
+    arguments.req_mem_per_core     = float(arguments.req_mem_per_core)
+
+    #if arguments.workspaceDir is None:
+        #arguments.workspaceDir = default_workspace
+        #if arguments.package_enables is None:
+            #arguments.package_enables = default_package_enables
+    #else:
+        #if arguments.package_enables is None:
+            #arguments.package_enables = os.path.join(arguments.workspaceDir, "packageEnables.cmake")
+
+
+    # Print the arguments to the console
+    print("\n")
+    print("+" + "-"*78 + "+")
+    print("| Parameters")
+    print("+" + "-"*78 + "+")
+    print("| - CONFIGURATION_FILE     : {configfile}".format(**vars(arguments)))
+    print("| - MODE                   : {mode}".format(**vars(arguments)))
+    print("| - REQ_MEM_PER_CORE       : {req_mem_per_core}".format(**vars(arguments)))
+    print("| - MAX_CORES_ALLOWED      : {max_cores_allowed}".format(**vars(arguments)))
+    print("| - NUM_CONCURRENT_TESTS   : {num_concurrent_tests}".format(**vars(arguments)))
+    print("|")
+    print("| - JOB_BASE_NAME          : {job_base_name}".format(**vars(arguments)))
+    print("| - WORKSPACE              : {workspaceDir}".format(**vars(arguments)))
+    print("|")
+    print("| - TRILINOS_SOURCE_REPO   : {sourceRepo}".format(**vars(arguments)))
+    print("| - TRILINOS_SOURCE_BRANCH : {sourceBranch}".format(**vars(arguments)))
+    print("|")
+    print("| - TRILINOS_TARGET_REPO   : {targetRepo}".format(**vars(arguments)))
+    print("| - TRILINOS_TARGET_BRANCH : {targetBranch}".format(**vars(arguments)))
+    print("|")
+    print("| - PULLREQUESTNUM         : {github_pr_number}".format(**vars(arguments)))
+    print("| - BUILD_NUMBER           : {job_number}".format(**vars(arguments)))
+    print("| - PACKAGE_ENABLES        : {package_enables}".format(**vars(arguments)))
+    print("|")
+    print("| - DRY_RUN                : {dry_run}".format(**vars(arguments)))
+    print("+" + "-"*78 + "+")
```

Comment:
Should the printing be pulled out into a separate routine?

Reply:
In this particular case I don't know that it matters either way.  Putting it into a function might make Uncle Bob happy but the purpose of this block is to print out the arguments that were just parsed.  In this case, not ALL the arguments but the ones that came in from the PR job's parameters itself.

I may expand this to dump out what the values of all the parameters are but I think it can stay here.

Reply:
2 reasons to create a function
- the code is shared between two routines (not applicable here)
- for organization - i.e. to make it more readable. This is where I and Bob Martin part ways. This is small enough to leave as is in my opinion.